tests/general/InvalidLinkBearTest.py

```diff
@@ -82,6 +82,7 @@ def test_run(self):
         valid_file = """
         http://httpbin.org/status/200
         http://httpbin.org/status/201
+        http://httpbin.org/status%20is/200
```

Comment:
Please add test cases along with the commit that needs these test cases :)
Squish both the commits together ?